docs/source/migrate_to_notebook7.md

```diff
@@ -1,27 +1,62 @@
 # Migrating to Notebook 7
 
-```{warning}
-The Jupyter Notebook application is undergoing a major refactoring to
-improve the user experience and to make it easier to maintain and extend.
+*Updated 2023-04-24*
 
-This set of migration guides are intended to help you migrate your Classic Notebook
-setup and extensions to the new Jupyter Notebook 7, which is built on top of JupyterLab 4.
+```{warning}
+The Jupyter Notebook application's newest version 7 upgrade might break your
+extensions or other customizations, please read more below to find out if you
+need to take any actions to ensure a smooth, uninterrupted experience
 ```
 
-## Sunrising the Jupyter Notebook 7
+A major upgrade to the Jupyter Notebook app is coming with Notebook 7! This
+upgrade will bring a heap of new features, but will also break backwards
+compatibility with many classic Notebook features and customizations.
+
+This set of guides is here to help you migrate your Classic Notebook setup and
+extensions to the new Notebook 7.
+
+## What you need to do
+
+For users who don't use extensions or other customizations, you will seamlessly
+receive the new Notebook 7 when you `pip install notebook` once version 7 is
+released out of beta, along with all its new features, like realtime
+collaboration, debugger, and theming.
+
+For users who need to use extensions or other customizations, you have a couple
+of options:
+
+- Look for Notebook 7 compatible versions of the extensions you already use,
+  and [find replacements for those that are not available]
+
+- If you NEED to maintain compatibility with the Classic Notebook for extensions
+  or other customizations that are critical to your workflows, you can switch to
+  [nbclassic], the new home for users who need to maintain compatibility with the
+  old notebook interface
+
+## Why things are changing
```

Comment:
Maybe something like the following?

```suggestion
## Why a new major version?

Reply:
I think this one is worth keeping, because it simplifies the language and reduces jargon (related to semantic versioning / "major version", which not everyone is going to know about). Maybe there's a plain-language compromise, like "Why the big version change?" Is it the "major-version-number-is-being-incremented-to-7" idea that you want to communicate here?

Reply:
@jtpio Is "Why the big version change?" good?

Reply:
Why not use "major"? "big" sounds ambiguous: what does big mean?

Otherwise maybe "Why a new version" could also work?

Reply:
Big was supposed to communicate that sweeping changes are taking place in plain-language (without the "major version" jargon and the odd-sounding/confusing phrasing that it necessitates...."Why a new major version" sounds odd, but "Why a major new version" does not...[probably related to points noted in this BBC article](https://www.bbc.com/news/blogs-trending-37285796)...).

I've pushed an update to just "Why a new version?" which I think is good enough, let me know if you've got any other suggestions.